ci/pipeline.yml

```diff
@@ -190,9 +311,26 @@ resources:
     type: git
     icon: github
     source:
-      uri: https://github.com/18F/federalist-proxy
+      uri: ((proxy-repository-url))
       branch: staging
 
+  - name: src-production
+    type: git
+    icon: github
+    source:
+      uri: ((proxy-repository-url))
+      branch: main
+
+  - name: pr
+    type: pull-request
+    check_every: 1m
+    source:
+      repository: 18F/federalist-proxy
```

Comment:
The GH PR resource only needs the `org/repo` path so do we hardcode or change the `((proxy-repository-url))` variable and make it compound like `((proxy-git-base-url))/((proxy-repository-path))`

Reply:
Hopefully we don't do another org change soon, but I'd say if it's not too much effort this would be the way to go

Reply:
Sounds good. _Note: There may be a point in the future where cloud.gov switches git servers._